iOS/immersive-reader-sdk/immersive-reader-sdk/ImmersiveReader.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  ImmersiveReader.swift
+//  immersive-reader-sdk
+//
+//  Created by Dilip Patidar on 19/11/19.
+//
+
```

Comment:
MSFT uses a standard file header, please use this header in files that you create:

// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.

Reply:
Done